ketcher-autotests/tests/Structure-Creating-&-Editing/Text-Tool/text-formating.spec.ts

```diff
@@ -0,0 +1,239 @@
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: '13' }).click();
+    await page.getByText('25', { exact: true }).click();
+    await page.getByRole('button', { name: 'bold' }).click();
+    await page.getByRole('button', { name: 'italic' }).click();
+    await page.getByRole('button', { name: 'superscript' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.getByText('TEST123').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'subscript' }).click();
+    await pressButton(page, 'Apply');
+  });
+
+  test('Text tool - Save as .ket file', async ({ page }) => {
+    // Test case: EPMLSOPKET-2235
+    // Checking if possible to put different styles on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEST321');
+    await pressButton(page, 'Apply');
+    await page.getByText('TEST321').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: '13' }).click();
+    await page.getByText('20', { exact: true }).click();
+    await page.getByRole('button', { name: 'bold' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await saveToFile('ketfile01.ket', 'ketFile');
+    await waitForRender(page, async () => {
+      await selectTopPanelButton(TopPanelButton.Clear, page);
+    });
+  });
+
+  test('Text tool - Open saved .ket file', async ({ page }) => {
+    await selectTopPanelButton(TopPanelButton.Open, page);
+    await openFromFileViaClipboard('tests/test-data/KET/ketfile01.ket', page);
+    await clickInTheMiddleOfTheScreen(page);
+    await takeEditorScreenshot(page);
+    await page.getByText('TEST321').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'italic' }).click();
+    await waitForRender(page, async () => {
+      await pressButton(page, 'Apply');
+    });
+  });
+
+  test('Text tool - Cut/Copy/Paste', async ({ page }) => {
+    // Test case: EPMLSOPKET-2272
+    // Checking if user is able to copy and paste the created text objects
+    const x = 250;
+    const y = 300;
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEXT001');
+    await pressButton(page, 'Apply');
+    await copyAndPaste(page);
+    await page.getByTestId('canvas').click({ position: { x, y } });
+  });
+
+  test('Text tool - Checking if user is able to cut and paste the created text objects', async ({
+    page,
+  }) => {
+    // Opening a file with created ealier text (task EPMLSOPKET-2272 ) and doing copy/paste action on it
+    await openFileAndAddToCanvas('KET/two-text-objects.ket', page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
```

Comment:
On line 191 you use 'waitForRender' but not here. Is this intended?

Reply:
I noticed that the screenshot comparison is flaky in the one of a test but not in the middle. That's why I use it just once.